### DISCUSSION TITLE
How to Manage Shared UI State

### DISCUSSION THREAD

**Discussion by emmatraversy** [inertiajs/inertia#1820] [Help]
Hi everyone.
I'm building a custom dashboard and I have this double tier collapsible navigation similar to sidebarOpen state.
but this state is going to affect other components of the ui. 

I was wondering what is the best way to share ui state between different components in inertia?
Is going to be conflicts with useContext or Zustand in inertia?

**Comment by ramonmalcolm10** [comment]
There shouldn’t be any conflict

**Reply by emmatraversy** [reply]
Thanks for your reply. I tested it yesterday and it worked fine.
But I was wondering if anyone else has experienced any issues with using inertia and zustand together so I asked it.

Did you use them both in a medium sized project? could you tell me about your experience?

**Reply by ramonmalcolm10** [reply]
Worked with it on a e-commerce site, didn’t experience any issue.
